extensions/amp-social-share/0.1/test/test-amp-social-share.js

```diff
@@ -175,8 +175,8 @@ describes.realWin('amp-social-share', {
       el.implementation_.handleClick_();
       expect(el.implementation_.win.open).to.be.calledOnce;
       expect(el.implementation_.win.open).to.be.calledWith(
-          'mailto:?subject=doc%20title&' +
-            'body=https%3A%2F%2Fcanonicalexample.com%2F',
+          'mailto:sample%40xyz.com?subject=doc%20title&' +
```

Comment:
Please add a second test that shows the email is removed if recipient is empty.

Reply:
Done